.github/workflows/docker.yml

```diff
@@ -56,24 +58,28 @@ jobs:
           created="$(date -u +'%Y-%m-%dT%H:%M:%SZ')"
           echo "Setting output: created=$created"
           echo "created=$created" >> "$GITHUB_OUTPUT"
+
       - name: Set up Docker Buildx
         id: buildx
         uses: docker/setup-buildx-action@v3
+
       - name: Available platforms
         run: echo ${{ steps.buildx.outputs.platforms }}
+
       - name: Login to DockerHub
         uses: docker/login-action@v3
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
+
       - name: Publish to Docker Hub
         uses: docker/build-push-action@v5
         with:
           context: .
           target: run
           build-args: |
             VERSION=${{ steps.prep.outputs.version }}
-          platforms: linux/amd64
+          platforms: linux/amd64,linux/arm64
```

Comment:
Is that really all we needed?

Reply:
I believe so, but I'm still testing it.  I will turn this into review mode if it succeeds.

Reply:
Does this need qemu defined to emulate arm64? I don't really know too much about the machine running this, but saw this in the documentation:

https://github.com/docker/build-push-action#usage

Reply:
I don't think so.  I believe the Figure team just wanted to be able to pull down an arm64 docker image.  We only had the amd64.  

I tested this by modifying the workflow to execute and publish from my branch https://hub.docker.com/layers/provenanceio/provenance/nullpointer0x00-1634-arm64-container-builds/images/sha256-e6189e7ab7d8b4181dc2c7fd996d3de062d633a789f625c70d1b998182a87322?context=explore. We can delete that image after this is merged.